lang/en.json

```diff
@@ -793,7 +793,7 @@
     "evidenceRequiredHalfFeeIncomeContent": "<p class=\"govuk-body\">If your gross annual income (your yearly income before tax) is less than £12,000, you can apply for a half fee.</p><p class=\"govuk-body\">You need to upload or send us recent proof of your gross annual income.</p>",
     "whereYourIncomeCanComeFrom": "Where your income can come from",
     "whereYourIncomeCanComeFromDetail": "<p class=\"govuk-body\">Your income can be from:</p><ul class=\"govuk-list govuk-list--bullet\"><li>employment</li><li>pensions</li><li>Universal Credit</li><li>Pensions Savings Credit</li><li>interest from savings and investments</li><li>the rent of property</li></ul><p class=\"govuk-body\">It can also come from non-means-tested benefits, such as:</p><ul class=\"govuk-list govuk-list--bullet\"><li>Attendance Allowance</li><li>Disability Living Allowance</li><li>Personal Independence Payment</li></ul>",
-    "evidenceRequiredHalfFeeEvidenceContent": "<h3 class=\"govuk-heading-m\">Supporting evidence of your income</h3><p class=\"govuk-body\">Your evidence can come from:</p><ul class=\"govuk-list govuk-list--bullet\"><li>paid employment – provide your P60 or 3 months’ consecutive wage slips from current employment</li><li>non-means-tested benefits and pensions – an official letter or notice</li><li>interest from capital, stocks, shares or bonds – statements or vouchers showing your gross income</li><li>self-employment – your most recent self-assessment tax return and HMRC tax calculation, or audited account certified by a qualified accountant</li></ul><p class=\"govuk-body\">Any evidence you send should show that the benefit or income is currently being paid to you. It should include your printed details such as your title and full name, address and postcode.</p><p class=\"govuk-body\">We do not accept bank statements.</p>",
+    "evidenceRequiredHalfFeeEvidenceContent": "<h2 class=\"govuk-heading-m\">Supporting evidence of your income</h2><p class=\"govuk-body\">Your evidence can come from:</p><ul class=\"govuk-list govuk-list--bullet\"><li>paid employment – provide your P60 or 3 months’ consecutive wage slips from current employment</li><li>non-means-tested benefits and pensions – an official letter or notice</li><li>interest from capital, stocks, shares or bonds – statements or vouchers showing your gross income</li><li>self-employment – your most recent self-assessment tax return and HMRC tax calculation, or audited account certified by a qualified accountant</li></ul><p class=\"govuk-body\">Any evidence you send should show that the benefit or income is currently being paid to you. It should include your printed details such as your title and full name, address and postcode.</p><p class=\"govuk-body\">We do not accept bank statements.</p>",
```

Comment:
no idea why this test has decided to just pop-up, but we did have h1>h3 so good spot from it

Reply:
https://github.com/ministryofjustice/opg-modernising-lpa/commit/15618a9ba7bc488a01343ed14c2503bb2bfc8ae5 this is why